src/app/pages/Designathons/Designathon24/components/About/index.jsx

```diff
@@ -0,0 +1,123 @@
+import cn from "./About.module.scss";
+
+import { useRef, useState, useEffect } from "react";
+import airplane from "../../assets/graphics/about/airplane.svg";
+import paperclip from "../../assets/graphics/about/paperclip.svg";
+import tr_gradient from "../../assets/graphics/about/tr_gradient.svg";
+import bl_gradient from "../../assets/graphics/about/bl_gradient.svg";
+import tr_stars from "../../assets/graphics/about/tr_stars.svg";
+import bl_stars from "../../assets/graphics/about/bl_stars.svg";
+
+const LINE_BREAKPOINTS = {
+	XL: 48,
+	LG: 40,
+	SM: 32,
+};
+
+const About = () => {
+	const [width, setWidth] = useState(0);
+	const [height, setHeight] = useState(0);
+	const [lines, setLines] = useState(0);
+	const [notecardLineHeight, setNotecardLineHeight] = useState(0);
+	const [notecardLineTop, setNotecardLineTop] = useState(0);
+
+	const textRef = useRef(null);
+
+	useEffect(() => {
+		const handleResize = () => {
+			if (textRef.current) {
+				const height = textRef.current.clientHeight;
+				setHeight(height);
+			}
+
+			if (window) {
+				const width = window.innerWidth;
+				setWidth(width);
+			}
+
+			const numLines = Math.floor(height / notecardLineHeight) - 1;
+			setLines(numLines > 0 ? numLines : 1);
+
+			/* Distance between notecard (blue) lines */
+			setNotecardLineHeight(
+				width >= 1280
+					? LINE_BREAKPOINTS.XL
+					: width >= 640
+						? LINE_BREAKPOINTS.LG
+						: LINE_BREAKPOINTS.SM,
+			);
+
+			/* Distance from top border of card */
+			setNotecardLineTop(width >= 640 ? 128 : 96);
```

Comment:
if you're setting all of these states at the exact same time every time they update, why not put it all into 1 state object so there's only ever 1 state?

Reply:
or.. better yet, can probably put all of these state changes into a separate component/separate hook